### DISCUSSION TITLE
UFC Undisputed 3

### DISCUSSION THREAD

**Discussion by Syrymka** [xenia-canary/game-patches#61] [Requests]
hello buddy can you make a patch for the game ufc undisputed 3

**Comment by bomabomaboma** [comment]
The issues section is for identifying problems with existing patches in the repository, not for requesting stuff.

**Comment by Margen67** [comment]
What sort of patch?

**Reply by luishn26** [reply]
A patch to make the game look faster from 10fps to 30fps or 60fps
